src/libraries/System.Runtime.Loader/tests/DefaultContext/System.Runtime.Loader.DefaultContext.Tests.csproj

```diff
@@ -2,8 +2,6 @@
   <PropertyGroup>
     <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
     <TestRuntime>true</TestRuntime>
-    <!-- LoadInDefaultContext test relies on no deps.json -->
```

Comment:
For reference, this was introduced in https://github.com/dotnet/corefx/pull/39279/ cc @ViktorHofer

Reply:
Seems that also modified Reflection tests which was  fixed by @steveharter in https://github.com/dotnet/runtime/commit/ac8ed2945e5ac0df7e0f42afd10c819dec0b166a

I bet there was a reason for it at the time, but most likely we've changed the test runner since then.  

I did a search for any other tests breaking this and didn't find any: https://github.com/search?q=repo%3Adotnet%2Fruntime+GenerateDependencyFile+path%3A*tests*csproj&type=code